docs/providers.md

```diff
@@ -38,10 +38,26 @@ The `java` provider takes following additional configuration options:
 
 * `bundles`: Path to extension bundles to enhance default Java language server's capabilities. See the [bundle](https://github.com/konveyor/java-analyzer-bundle) Konveyor uses.
 * `workspace`: Path to directory where the provider generates debug information such as logs.
+* `location`: Path to the source code of the application to analyze. The location can be a binary jar, war, or ear file.
+
+Here's an updated example config for the Java provider in `provider_settings.json`:
+
+```json
```

Comment:
this is changed so I wonder if we need to update this for the new settings.

Also changing again w/ the multi-init config change. do you think that we can get this in first and then do the API review and I can work on the rebase